crates/ef-testing/src/models/case.rs

```diff
@@ -107,7 +109,14 @@ impl BlockchainTestCase {
                 | "CallTheContractToCreateEmptyContract.json" // ef-test #115
                 | "OutOfGasContractCreation.json" // ef-test #116
                 | "CallContractToCreateContractAndCallItOOG.json" // ef-test #117
-        )
+                | "undefinedOpcodeFirstByte.json" // ef-tests #121
+                | "measureGas.json" // ef-tests #122
+                | "badOpcodes.json" // ef-tests #123
+                | "operationDiffGas.json" // ef-tests #124
+                | "invalidDiffPlaces.json" // ef-tests #125
+                | "invalidAddr.json" // ef-tests #126
+        ) || matches!(name, name if name.starts_with("opc") && name.ends_with(".json"))
```

Comment:
maybe we can create an issue about having two distinct places to add skips, by specific case or by suite.

Reply:
wdym

Reply:
ohh skipping a whole suite?

Reply:
kk, talked with greg and it seems we arent distinguishing case/suite rn for skips (case is a specific key/val pair within a json file, json file is a suite). what i meant was we'd have a `skipped_by_name` and a `skipped_by_regex` so its less of a scary big list to manage.

Reply:
ah i see

so case: key/val pair within a json
suite: json file
folder of tests: ?

Reply:
you're right! let's create the issue and improve clarity